src/windows/functions.cpp

```diff
@@ -403,7 +434,7 @@ MOUNTUTILS_RESULT Eject(ULONG deviceNumber) {
   }
 
   while (logicalDrivesMask) {
-    if (logicalDrivesMask & 1) {
+    if (logicalDrivesMask & 1 && IsDriveLetterEjectable(currentDriveLetter)) {
```

Comment:
IIRC in an earlier version of this PR, you checked that `currentDriveLetter` was still present in the bitmask returned by `GetLogicalDrives`. Would it be still worth doing that, or does `GetDriveType` now handle that for us automatically?